src/__tests__/prove.ts

```diff
@@ -38,9 +40,20 @@ describe('Prove endpoint', () => {
     const message = 'Signature for SealHub'
     const wallet = Wallet.createRandom()
     const signature = await wallet.signMessage(message)
+    const input = await generateInput(signature, message)
+    const data = {} as {
+      TPreComputes: string
+      U: string
+      s: string
+    }
+    Object.entries(input).forEach(([key, value]) => {
+      data[key as keyof typeof data] = JSONbig.stringify(value)
+    })
     await request(server)
       .post('/prove')
-      .send({ signature: signature, message })
+      .attach('U', Buffer.from(data.U))
```

Comment:
Maybe we can simplify each `.attach`, not sure

Reply:
No, we can't